.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,111 @@
+---
+name: Bug report
+about: libcoap crashes, produces incorrect output, or has incorrect behavior
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+----------------------------- Delete Below -----------------------------
+
+INSTRUCTIONS
+============
+
+Before submitting a new issue, please follow the checklist and try to find the
+answer.
+
+- [ ] I have read the documentation [libcoap Modules Documentation](https://libcoap.net/doc/reference/develop/modules.html)
+and the issue is not addressed there.
+- [ ] I have read the documentation [libcoap Manual Pages](https://libcoap.net/doc/reference/develop/manpage.html)
+and the issue is not addressed there.
+- [ ] I have updated my libcoap branch (develop) to the latest version and
+checked that the issue is present there.
+- [ ] I have searched the [Issue Tracker](https://github.com/obgm/libcoap/issues)
+(both open and closed - overwrite `is:issue is:open`) for a similar issue and
+not found a similar issue.
+- [ ] I have checked the [Wiki](https://github.com/obgm/libcoap/wiki) to see if
+the issue is reported there.
+- [ ] I have read the HOWTOs provided with the source.
+- [ ] I have read the [BUILDING](https://raw.githubusercontent.com/obgm/libcoap/develop/BUILDING)
+on how to build the source.
```

Comment:
"build the source" → "build from source"

Reply:
Updated